cmd/cluster-init/testdata/ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml

```diff
@@ -0,0 +1,45 @@
+periodics:
```

Comment:
What's this doing in here?

Reply:
ah it shouldn't be. it came over in the rebase.